easyvote/src/main/java/Logic/VotingTopic.java

```diff
@@ -20,5 +20,14 @@
   // Used to determine if the topic passed or not, according to the votes
   private boolean passed;
 
+  public VotingTopic() {
+	  //A random reference - example
+	  String type = TypeOfVote.ROP.toString();
```

Comment:
Can you please remove the example from the code, and add it again as a comment inside the class? I know this is a minor change, but we should keep the code clean ;-)

Reply:
Of course